ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -123,6 +123,9 @@ def main(args):
     ebm_stats['ebm_trials_figure'] = \
         f'https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/{args.output_figure}.svg'
     
+    # Tabulate number of trials for pharmaceuticals of interest
+    ebm_stats['tocilizumab_ct'] = trials_df['intervention'].str.contains('tocilizumab', case=False).sum()
```

Comment:
```suggestion
    ebm_stats['ebm_tocilizumab_ct'] = trials_df['intervention'].str.contains('tocilizumab', case=False).sum()
```

Your approach is correct, and this should work.  I'm using the convention where the variable prefix (`ebm_`) denotes the data source to help with later debugging if needed.